lib/broccoli/fastboot-config.js

```diff
@@ -104,7 +104,9 @@ FastBootConfig.prototype.readAssetManifest = function() {
     var assetMap = JSON.parse(fs.readFileSync(assetMapPath));
     return assetMap;
   } catch (e) {
-    // No asset map was found, proceed as usual
+    if (assetMapPath) {
```

Comment:
This warning (especially after taking care of @danmcclain's feedback) will only trigger if asset map generation is enabled and no asset map was found. I think that is absolutely a scenario that warrants a warning.